db/migrations/007_gate_encrypted_answers.sql

```
@@ -0,0 +1,40 @@
+    ciphertext     TEXT        NOT NULL,              -- ASCII-armored age output, never plaintext
+    skipped        BOOLEAN     NOT NULL DEFAULT FALSE,
+    created_at     TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+
+    -- One row per (session, question). Re-answering updates in place rather
+    -- than accumulating history, so a visitor changing their mind does not
+    -- leave the superseded answer behind.
+    UNIQUE (session_id, question_index)
+);
+
+CREATE INDEX IF NOT EXISTS idx_gate_session
+    ON gate_encrypted_answers (session_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)